packages/admin/admin-rte/src/core/makeRteApi.ts

```diff
@@ -34,7 +35,11 @@ function defaultFormatContent(v: ContentState): any {
 }
 
 function makeRteApi<T = any>(o?: IMakeRteApiProps<T>) {
-    const { decorators = [LinkDecorator], parse = defaultParseContent, format = defaultFormatContent }: IMakeRteApiProps<T> = o || {};
+    const {
+        decorators = [LinkDecorator, NonBreakingSpaceDecorator],
```

Comment:
I'm not sure if its a good idea that I add the decorator here as all non-breaking spaces would be automatically replaced by `[+]` in the editor. If I don't add it here, all decorators need to be explicitly set when creating the rte api. 

I could also only show the `[+]` when non-breaking space is added to `supports`

Reply:
can you insert nbsp without the supports? (copy paste I assume will work)

Then it is ok to show it always imho.

Reply:
I just tried it: it still works when I remove the option from the supports. So I guess we may add it here all the time.

Reply:
Done here: 315276860ba24248b22c9aacb47063e1bc0a202d